### PR TITLE
allow peridot project to specify a build pool type in additional to build pool architecture

### PR DESCRIPTION
**Pull request by PaulCharlton**
1) allow peridot project to specify a build pool type in additional to build pool architecture
2) do not place user specific IntelliJ workspace artifacts into git

### CODE REVIEW

File: peridot/db/psql/project.go [hunk 1]
```diff
@@ -264,6 +268,7 @@ func (a *Access) UpdateProject(id string, project *peridotpb.Project) (*models.P
 		TargetVendor:       project.TargetVendor,
 		AdditionalVendor:   project.AdditionalVendor.Value,
 		Archs:              project.Archs,
+		BuildPoolType:      project.BuildPoolType,
```
Comment (mstg) [inline peridot/db/psql/project.go:271]: If you just change this to `project.BuildPoolType.Value`, then we should be all set

**Review by mstg — CHANGES_REQUESTED** [review]

File: peridot/db/psql/project.go (hunk 1 above)
Comment (PaulCharlton) [reply peridot/db/psql/project.go]: updated

**Comment by mstg** [comment]
Thank you!

/approve
/lgtm

**Review by mstg — APPROVED** [review]

**Pull request merged**